rust/smoketest/README.md

```diff
@@ -36,14 +38,13 @@ All settings for running the smoke-tests are set via environmental variables. Th
 # set env variables
 export CLIENT_CERT=/tmp/cert.pem
 export CLIENT_KEY=/tmp/key.pem
-export OPENVASD=192.168.0.1:3000
```

Comment:
In the table above is still `OPENVASD`

Reply:
I have updated the var name in the table